### PR TITLE
Just start stubbing in things for feedback

### PR DESCRIPTION
**Pull request by montgomp**
### Summary of changes 🔍 
Starting to stub out bits and pieces to have some skeleton to start building on.

### Testing 🧪
 > [ Add any special instructions needed to test or view your changes such as environment URLs or other config details. ]

### Checklist ✔️
- [ ] Linked associated issue (if present)
- [ ] Added relevant labels
- [ ] Requested developer reviews
- [ ] Request UI review from design / PM
- [ ] Verify all code checks are passing

### CODE REVIEW

File: src/Apps/Legion/Contexts/AppDataContext/AppDataContext.types.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { IExtensionClient } from '../../EntryPoints/LegionApp/ShellTypes';
```
Comment (msnyder-msft) [inline src/Apps/Legion/Contexts/AppDataContext/AppDataContext.types.ts:1]: What would you think about having another "App" called "Core" or "Shared" and putting stuff like this in there?

File: src/Apps/IndustrialMetaverse/Site/Site.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { ISiteProps } from './Site.types';
+
+const Site: React.FC<ISiteProps> = ({ extensionClient }) => {
+    if (extensionClient) {
+        return <div>Site</div>;
+    } else {
+        return <div>Site without client</div>;
```
Comment (msnyder-msft) [inline src/Apps/IndustrialMetaverse/Site/Site.tsx:8]: great idea.

**Review by msnyder-msft — APPROVED** [review]

File: src/Apps/IndustrialMetaverse/Site/Site.tsx (hunk 2 above)
Comment (montgomp) [reply src/Apps/IndustrialMetaverse/Site/Site.tsx:8]: We should be able to leverage react-router-dom hook `useLocation` to get any pathing and query string parameters to pass from here into the various sub-explorers

File: src/Apps/IndustrialMetaverse/Site/KPIExplorer/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+//Not really sure what all needs to go here yet
```
Comment (msnyder-msft) [inline src/Apps/IndustrialMetaverse/Site/KPIExplorer/index.ts:1]: can we keep the same folder structure as we have in the LegionApp where it's got the Adapter, COmponent, Contexts folders under the root?

Comment (msnyder-msft) [reply src/Apps/IndustrialMetaverse/Site/KPIExplorer/index.ts:1]: image

Comment (montgomp) [reply src/Apps/IndustrialMetaverse/Site/KPIExplorer/index.ts:1]: We have Legion and IM apps. IM only has Site (I could move it to an EntryPoints folder). Site has the two main "explorers" which could have Contexts/Adapters/Hooks of their own or have shared Adapters/Contexts/Hooks in Site. I definitely need to move any .tsx into Components folders and .types.ts into Models.

**Review by msnyder-msft — APPROVED** [review]

**Pull request merged**